geozero/src/mvt/mvt_reader.rs

```diff
@@ -208,14 +209,14 @@ fn process_polygon<P: GeomProcessor>(
 
     for (i, ring) in rings.iter().enumerate() {
         if ring[0] != CommandInteger::from(Command::MoveTo, 1) {
-            return Err(GeozeroError::GeometryFormat);
+            return Err(GeozeroError::MvtError(MvtError::GeometryFormat));
```

Comment:
I'm not too certain, but it **might** be possible to simplify it to this (don't accept it without checking first) - basically because you have declared `from` attribute, so in theory it should call the `into()` on the error?
 
```suggestion
            Err(MvtError::GeometryFormat)?
```

Reply:
Yeah, it felt weird, I'd forgotten I could use `.into()` for the conversion. I'll check but I'm pretty certain there's a better approach!

Reply:
You were right, I could just call the `?` operator. :facepalm: